### PR TITLE
Fixes decal painter can paint walls

### LINKED ISSUE #6603: Decal painter can paint walls
image

Decal painter tool can paint really obnoxious decals as well as walls.

### PR DESCRIPTION
**Pull request by Mat05usz**
# About The Pull Request

Fixes #6603

Also removes the ugly and obnoxious full tile striped decal.

## Why It's Good For The Game

Bad code bad

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

## Changelog
:cl:

fix: Decal painter can no longer paint on walls
remove: Removed ugly and obnoxious full tile striped decal from the decal painter

/:cl:

### CODE REVIEW

File: code/game/objects/items/airlock_painter.dm [hunk 1]
```diff
@@ -165,16 +165,23 @@
...
 	. = ..()
 	if(!proximity)
 		to_chat(user, "<span class=notice>You need to get closer!")
 		return
-	if(use_paint(user) && isturf(target))
+	if(isturf(target))
 		var/turf/open/floor/F = target
+
+		if(!istype(F))
+			to_chat(user, "<span class=notice>Only floors can be painted!")
```
Comment (PowerfulBacon) [inline code/game/objects/items/airlock_painter.dm]: Close the span with </span>

Comment (PowerfulBacon) [inline code/game/objects/items/airlock_painter.dm]:
```suggestion
			to_chat(user, "<span class=notice>Only floors can be painted!</span>")
```

Comment (Mat05usz) [reply code/game/objects/items/airlock_painter.dm]: Me dum didn't even notice that

Comment (Mat05usz) [reply code/game/objects/items/airlock_painter.dm]: Me when copy and paste. Done.

Comment (Mat05usz) [reply code/game/objects/items/airlock_painter.dm]: Done

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**